.github/issue_templates/create-a-bug-report.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
should the directory be `ISSUE_TEMPLATE` instead of `issue_templates`, like [here](https://github.com/uc-cdis/gen3-community/tree/master/.github/ISSUE_TEMPLATE)?